docker-entrypoint

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 set -e
-if [ -d "$HOME/.venv/bin" ]; then
-    . ~/.venv/bin/activate
-fi
+
+# Enable SSH and give it access to app setting env variables
+if [[ "$ENABLE_SSH" = "true" ]]; then
+    service ssh start
+    eval $(printenv | sed -n "/^PWD=/!s/^\([^=]\+\)=\(.*\)$/export \1=\2/p" | sed 's/"/\\\"/g' | sed '/=/s//="/' | sed 's/$/"/' >> /etc/profile)
 
```

Comment:
Maybe missing 'fi' on line 8? 

./docker-entrypoint: line 5[6](https://github.com/City-of-Turku/parkkihubi/actions/runs/15603489132/job/43947686917#step:7:7): syntax error: unexpected end of file